content/02.main-text.md

```diff
@@ -0,0 +1,41 @@
+## Introduction {.page_break_before}
+
+## Methods {.page_break_before}
+
+## Results {.page_break_before}
+
+For each of the 125 driver genes from the Vogelstein et al. 2013 paper, we trained models to predict mutation status (presence or absence) from RNA-seq data, derived from the TCGA Pan-Cancer Atlas.
+For each optimizer, we trained LASSO logistic regression models across a variety of regularization parameters (see Methods for parameter range details), for 4 cross-validation splits x 2 replicates (random seeds) for a total of 8 different models per parameter.
+Cross-validation splits were stratified by cancer type.
+
+Figure {@fig:optimizer_compare_mutations}A shows performance vs. model complexity for prediction of KRAS mutation status using each optimizer.
+Previous work has shown that pan-cancer classifiers of Ras mutation status are accurate and biologically informative [@doi:10.1016/j.celrep.2018.03.046].
+As model complexity increases (more nonzero coefficients) for the `liblinear` optimizer, we observe that performance increases then decreases, corresponding to overfitting for high model complexities/numbers of nonzero coefficients.
+On the other hand, for the SGD optimizer, we observe an increase in performance as model complexity increases, with models having no nonzero coefficients performing the best.
```

Comment:
I think you might want to make this an A/B/C figure instead of A/B so you could add a pointer to the SGD panel here.